src/main/java/com/box/sdk/BoxResourceIterable.java

```diff
@@ -68,28 +73,23 @@
      *            the maximum number of items to return in a page
      */
     public BoxResourceIterable(BoxAPIConnection api, URL url, int limit) {
-        this.api = api;
-        this.url = url;
-        this.limit = limit;
-        this.iterator = new IteratorImpl(null);
+        this(api, url, limit, null, null);
     }
 
     /**
-     * Constructor.
+     * Construtor.
```

Comment:
Constructor

Reply:
How on Earth did you notice this?!?!  
Done!